### PR TITLE
Remove deprecated usage in the request files

### PR DESCRIPTION
**Pull request by sbrunner**
Depends on #463

### CODE REVIEW

File: core/src/test/resources/org/mapfish/print/processor/map/center_osm_fixedscale/requestData.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         }, {
           "type": "osm",
           "baseURL": "http://center_osm_fixedscale.osm:1234",
-          "imageFormat": "tiff"
+          "imageExtension": "tiff"
```
Comment (tsauerwein) [inline core/src/test/resources/org/mapfish/print/processor/map/center_osm_fixedscale/requestData.json]: There is a type in the commit message ("deprecated").

**Comment by tsauerwein** [comment]
Thanks!

Do you want to set up a linter for the JSON files?

**Comment by sbrunner** [comment]
@tsauerwein it can be a good idea, but I don't know well Gradle, then it not easy for me to do it...

File: core/src/test/resources/org/mapfish/print/processor/map/center_osm_fixedscale/requestData.json (hunk 1 above)
Comment (sbrunner) [inline core/src/test/resources/org/mapfish/print/processor/map/center_osm_fixedscale/requestData.json]: Thanks, fixed :-)

**Pull request merged**